app/helpers/filter_dropdown_helper.rb

```diff
@@ -99,6 +99,11 @@ def dropdown_toggler_options
 
   def translate_value(filter_attribute, t_scope)
     return t('all') if filter_attribute.blank?
-    t(filter_attribute, t_scope)
+    unless filter_attribute.class == Department ||
```

Comment:
```ruby
if your_condition
  return filter_attribute.to_s
end
t(filter_attribute, t_scope)

Reply:
And why arent you checking for `filter_attribute.is_a?` ?
If its a symbol or a model. You can check for that. And in that case your condition will be shorter and it will be more generic

Reply:
I had tried that. But when filtering for departments (and categories I guess), it first checks if there is translation for the department name and then for the department id.